### PR TITLE
aiac: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.es/osx/aiac.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# aiac
+
+> Utiliza OpenAI para generar configuraciones IaC, utilidades, consultas y más.
+> Más información: <https://github.com/gofireflyio/aiac>.
+
+- Genera Terraform para una cuenta de almacenamiento Azure:
+
+`aiac get terraform {{para una cuenta de almacenamiento azure}}`
```
Comment (navarroaxel) [inline pages.es/osx/aiac.md:8]: This application works with Spanish specs or English-only?

**Review by blueskyson — APPROVED** [review]

File: pages.es/osx/aiac.md (hunk 1 above)
Comment (kant) [reply pages.es/osx/aiac.md:8]: English only

Comment (kant) [reply pages.es/osx/aiac.md]: applied english queries

**Comment by kant** [comment]
[Codespell](https://github.com/tldr-pages/tldr/actions/workflows/codespell.yml)

**Pull request merged**